llarp/config/key_manager.cpp

```diff
@@ -61,7 +61,7 @@ namespace llarp
     m_lokidRPCPassword = config.lokid.lokidRPCPassword;
 
     RouterContact rc;
-    bool exists = rc.Read(m_rcPath.c_str());
+    bool exists = rc.Read(m_rcPath.string().c_str());
```

Comment:
I think this would be nicer to just have `RouterContact::Read` take a `fs::path` instead of a c-str instead (and then have it do the path -> c-string conversion if needed).